redis/tests/test_cluster_async.rs

```diff
@@ -2183,6 +2184,81 @@ fn test_async_cluster_periodic_checks_update_topology_after_failover() {
     .unwrap();
 }
 
+#[test]
+fn test_async_cluster_recover_disconnected_management_connections() {
+    // This test aims to verify that the management connections used for periodic checks are reconnected, in case that they get killed.
+    // In order to test this, we choose a single node, kill all connections to it which aren't user connections, and then wait until new
```

Comment:
Why don't you just kill the management connection?

Reply:
https://github.com/amazon-contributing/redis-rs/pull/56/files#diff-85259ca2185589cf8979102fc91c6ce1a9461aac8ed605472b9ad53c3324c400R2111